code/modules/mob/living/carbon/xenomorph/castes/boiler/boiler.dm

```diff
@@ -14,6 +14,8 @@
 	upgrade = XENO_UPGRADE_ZERO
 	gib_chance = 100
 	drag_delay = 6 //pulling a big dead xeno is hard
+	var/corrosive_ammo = 0
```

Comment:
Can you put these on the ability datum instead?

Reply:
I don't want to limit the amount of each globule, I only want to limit the total amount.

Reply:
Mhm. Still, the ability datum can track this.